### PR TITLE
[IMP] open_academy: Relation between courses and sessions T#59013

### PR DESCRIPTION
**Pull request by geraspace**
Hi @edy1192, I just pushed some changes. Could you check them? Thank you!

### CODE REVIEW

File: open_academy/models/open_academy_session.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    _name = "open.academy.session"
+
+    name = fields.Char(required=True)
+    start_date = fields.Date()
+    duration = fields.Float(digits=(6,2), help="Duration in days")
+    seats = fields.Integer(string="Number of seats")
+
+    instructor_id = fields.Many2one("res.partner")
+    course_id = fields.Many2one("open.academy.course", ondolete="cascade", required=True)
+    attende_id = fields.Many2many("res.partner", string="Attendes")
```
Comment (edy1192) [inline open_academy/models/open_academy_session.py]:
```suggestion
    attendee_ids = fields.Many2many("res.partner", string="Attendes")
```

See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields) for fields

File: open_academy/models/open_academy_session.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+class OpenAcademySession(models.Model):
+    _name = "open.academy.session"
+
+    name = fields.Char(required=True)
+    start_date = fields.Date()
+    duration = fields.Float(digits=(6,2), help="Duration in days")
+    seats = fields.Integer(string="Number of seats")
+
+    instructor_id = fields.Many2one("res.partner")
+    course_id = fields.Many2one("open.academy.course", ondolete="cascade", required=True)
```
Comment (edy1192) [inline open_academy/models/open_academy_session.py]:
```suggestion
    course_id = fields.Many2one("open.academy.course", ondelete="cascade", required=True)
```

File: open_academy/models/open_academy_course.py [hunk 3]
```diff
@@ -7,4 +6,7 @@ class OpenAcademyCourse(models.Model):
     _description = "open.academy.course.description"
 
     name = fields.Char(string="Course", required=True)
-    description = fields.Text(string="Course description", required=True)
+    description = fields.Text(required=True)
+
+    responsible_id = fields.Many2one('res.users')
+    session_id = fields.One2many('open.academy.session', 'course_id')
```
Comment (edy1192) [inline open_academy/models/open_academy_course.py]:
```suggestion
    session_ids = fields.One2many('open.academy.session', 'course_id')
```
See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields) for fields

File: open_academy/views/open_academy_course_views.xml [hunk 4]
```diff
@@ -33,10 +34,21 @@
...
                         <page string="Description">
-                            <field name="description"/>
+                            <group>
+                                <field name="description"/>
+                            </group>
+                        </page>
+                        <page string="Sessions">
+                            <field name="session_id">
+                                <tree>
+                                    <field name="name" />
```
Comment (edy1192) [inline open_academy/views/open_academy_course_views.xml:49]: Show all the fields of `open.academy.session` model, except attendees

File: open_academy/views/open_academy_session_views.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="open_academy_session_action" model="ir.actions.act_window">
+        <field name="name">Sessions</field>
+        <field name="res_model">open.academy.session</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
```
Comment (edy1192) [inline open_academy/views/open_academy_session_views.xml:9]: Add a tree and form view for `open.academy.session` model please.

**Review by edy1192 — CHANGES_REQUESTED** [review]
Do you need the "update" commit?

**Comment by edy1192** [comment]
@geraspace  change the PR's and commit title. This is an improvement use the IMP tag please.

**Comment by geraspace** [comment]
I think It's done, @edy1192. Thanks!

**Review by edy1192 — APPROVED** [review]

**Comment by edy1192** [comment]
> @geraspace change the PR's and commit title. This is an improvement use the IMP tag please.

You need to change the commit's title, use an IMP tag

**Review by edy1192 — CHANGES_REQUESTED** [review]
Change the commit's title please

**Review by edy1192 — APPROVED** [review]

**Pull request merged**